### ISSUE TITLE
Can we skip loading the js entirely in test environment?

### ISSUE THREAD

**Issue by githorse** [ambethia/recaptcha#173]
Recaptcha verification is disabled by default in the test environment; however, the recaptcha tags themselves are still downloaded from Google. Since I have a ton of tests that load the page with the recaptcha on it, this significantly slows down my test suite (for no reason, since we're skipping verification anyway). I can just explicitly skip the recaptcha load in the view with `recaptcha_tags unless Rails.env.test?`, but that's a code smell in my book. Is there a better way to accomplish this?

**Comment by grosser** [comment]
There might be people testing/asserting that the inputs is present or something like that ... but I think it's the saner default to not show them ... can you open a PR to change that ?

Alternative code would be:

```
recaptcha_tags  unless Recaptcha::Verify.skip?(Rails.env)
```

**Comment by githorse** [comment]
Yeah, I wondered if it should still render a hidden field or an empty div or something to test against ... I will look into doing a PR for this.

**Comment by programatologist** [comment]
Is this fixed? Can I look into and make a PR otherwise?

**Comment by githorse** [comment]
@programatologist I got distracted and never worked on this (can't remember at the moment what my workaround was). As far as I know it could still use a PR. @grosser?
